### ISSUE TITLE
Remove the unused condition from lessons_completed_chapter_view.xml for Accessibility 

### ISSUE THREAD

**Issue by MohitGupta121** [oppia/oppia-android#4888]
Currently we using `viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES` condition ForAccessibility which is always True in `lessons_completed_chapter_view.xml` , So we can remove this condition and directly use the true value.
